api-test/src/main/resources/idRepository/AuthInternalLock/AuthInternalLock.yml

```diff
@@ -2,6 +2,7 @@ AuthInternalLock:
   IdRepository_AuthInternalLock_Uin_Demo_Smoke_Pos:
       endPoint: /idrepository/v1/identity/authtypes/status
       description: Lock UIN for the auth type demo
+      uniqueIdentifier: TC_Authlock_UIN_01
```

Comment:
Unique identifier format should be TC_moduleName_testScenario_01

For example TC_IDRepo_AuthInternalLock_01